### DISCUSSION TITLE
Low-cost RPi Zero-based endoscope camera

### DISCUSSION THREAD

**Discussion by billz** [RaspAP/raspap-webgui#823] [Show and tell]
Hot off the press...  with a nice attribution to RaspAP, which provided wireless connectivity for the prototype!

> ![](https://www.raspberrypi.org/app/uploads/2021/03/Screenshot-2021-03-11-at-17.40.48.png)
> This Raspberry Pi-powered wireless option also saves thousands of dollars. It was built for just $230, whereas contemporary wired options cost around $28,000. Urologists at the University of Cape Town created the prototype. J. M. Lazarus & M. Ncube hope their design will be more accessible to medical settings that have less money available. You can read their [research paper](https://afju.springeropen.com/articles/10.1186/s12301-021-00127-z) for an in-depth look at the whole process.

https://www.raspberrypi.org/blog/low-cost-raspberry-pi-zero-endoscope-camera/

**Comment by whi4** [comment]
This is pretty cool.
